inference-engine/src/mkldnn_plugin/CMakeLists.txt

```diff
@@ -202,6 +207,11 @@ target_include_directories(${TARGET_NAME}_obj PRIVATE $<TARGET_PROPERTY:inferenc
 
 set_ie_threading_interface_for(${TARGET_NAME}_obj)
 
+if(SELECTIVE_BUILD STREQUAL "ON")
+    # After disabling a block of code, some variables might be unused.
+    target_compile_options(${TARGET_NAME}_obj PRIVATE -Wno-unused-variable)
+endif()
```

Comment:
do we need to use conditional compilation for `${TARGET_NAME}_obj`? Looks like static version can be built as is..

Reply:
fixed

Reply:
We also have:
```
$<TARGET_PROPERTY:openvino::conditional_compilation,INTERFACE_INCLUDE_DIRECTORIES>
```
For obj library